default/python/universe_generation/empires.py

```diff
@@ -98,10 +128,11 @@ def compile_home_system_list(num_home_systems, systems):
     min_jumps = min(10, max(int(float(len(systems)) / float(num_home_systems * 2)), 5))
     # try to find the home systems, decrease the min jumps until enough systems can be found, or the min jump distance
     # gets reduced to 0 (meaning we don't have enough systems to choose from at all)
+    additional_reqs = homesystem_vicinity_filter if _FILTER_HOMESYSTEMS_BY_VICINITY else None
```

Comment:
As mentioned above, 
> though I expect it makes it more likely to drop down to a smaller min-jumps separation than it would have to otherwise.
Iv'e made a couple other changes in the last few days to streamline the creation of large and very large galaxies, which can take quite a while; one specifically about avoiding running extra cycles through rejected min-jump distances.  Additionally, even for smaller galaxies, the effect on min-jump may not be acceptable.  So I think this behavior should be a configurable choice going forward.